addons/main/script_macros_common.hpp

```diff
@@ -365,13 +365,34 @@ Macro: RETNIL()
 Example:
     (begin example)
         // _var is undefined
-        hintSilent format ["_var=%1", RETNIL(_var) ]; // "_var=any"
+        hintSilent format ["_var=%1", RETNIL(_var)]; // "_var=any"
     (end example)
 
 Author:
     Alef (see CBA issue #8514)
 ------------------------------------------- */
-#define RETNIL(VARIABLE) if (isNil{VARIABLE}) then {nil} else {VARIABLE}
+#define RETNIL(VARIABLE) ([VARIABLE] param [0,nil])
```

Comment:
Otherwise, maybe this is the fastest:

`(if !(isNil {VARIABLE}) then {VARIABLE})`

But I guess that depends on the expression of `VARIABLE`